apps/authentication-portal/src/main/webapp/consent.jsp

```diff
@@ -51,6 +51,11 @@
     <% } %>
 </head>
 <body class="login-portal layout authentication-portal-layout">
+
+    <% if (new File(getServletContext().getRealPath("util/timeout.jsp")).exists()) { %>
+    <jsp:include page="util/timeout.jsp"/>
+    <% } %>
```

Comment:
IMO, we should provide a WUM safe way to override/disable the timeout. @yasinmiran WDYT?

Reply:
+1 This has been addressed in commit d48d227. Now `timeout.jsp` file refer to files in `util` directory relative to its path. Tested this and the only caveat is if the external dev wants to wrap the `timeout.jsp` in another include directive, the rendering engine will result in an infinite loop. It seems JSP doesn't like relative paths.

Reply:
Also see changes in e8c7c49

Reply:
Nice (y)